### PR TITLE
 GG-29111: Added spec and scripts for building ODBC RPM

### PR DESCRIPTION
**Pull request by isapego**

### CODE REVIEW

File: modules/platforms/cpp/odbc/install/gridgain-odbc.spec [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+DriverODBCVer=03.00' \
+> %{_tmppath}/%{name}-%{version}-%{release}.ini
+
+odbcinst -i -d -f %{_tmppath}/%{name}-%{version}-%{release}.ini -v
+
+%preun
+odbcinst -u -d -n "Apache Ignite" -v
+
+%changelog
+* Tue May 19 2020 Igor Sapego <isapego@gridgain.com> - 8.7.17-1
```
Comment (vveider) [inline modules/platforms/cpp/odbc/install/gridgain-odbc.spec:104]: Change log should contain Organisation's name and support email, not personal.

File: pom.xml [hunk 2]
```diff
@@ -754,6 +754,14 @@
                                             </fileset>
                                         </replaceregexp>
 
+                                        <replaceregexp byline="true" encoding="UTF-8">
```
Comment (vveider) [inline pom.xml:757]: This script should ADD new changelog entry — date-time, author, version, changes and so on.

File: modules/platforms/cpp/odbc/install/gridgain-odbc.spec [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+DriverODBCVer=03.00' \
+> %{_tmppath}/%{name}-%{version}-%{release}.ini
+
+odbcinst -i -d -f %{_tmppath}/%{name}-%{version}-%{release}.ini -v
+
+%preun
+odbcinst -u -d -n "Apache Ignite" -v
+
+%changelog
+* Fri May 29 2020 Igor Sapego <info@gridgain.com> - 8.7.19-1
```
Comment (vveider) [inline modules/platforms/cpp/odbc/install/gridgain-odbc.spec:104]: Name should be `GridGain Systems`
Email — `support@gridgain.com`

File: modules/platforms/cpp/odbc/install/gridgain-odbc.spec [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+DriverODBCVer=03.00' \
+> %{_tmppath}/%{name}-%{version}-%{release}.ini
+
+odbcinst -i -d -f %{_tmppath}/%{name}-%{version}-%{release}.ini -v
+
+%preun
+odbcinst -u -d -n "Apache Ignite" -v
+
+%changelog
+* Fri May 29 2020 GridGain Systems <info@gridgain.com> - 8.7.19-1
```
Comment (vveider) [inline modules/platforms/cpp/odbc/install/gridgain-odbc.spec:104]: I suggest to set support@gridgain.com as mail.

File: pom.xml (hunk 2 above)
Comment (isapego) [reply pom.xml:762]: Added scripts that will add changelog entry below

File: modules/platforms/cpp/odbc/install/gridgain-odbc.spec (hunk 4 above)
Comment (isapego) [reply modules/platforms/cpp/odbc/install/gridgain-odbc.spec]: Done

File: modules/platforms/cpp/odbc/install/gridgain-odbc.spec (hunk 3 above)
Comment (isapego) [reply modules/platforms/cpp/odbc/install/gridgain-odbc.spec]: Fixed

**Pull request merged**